fork.js

```diff
@@ -0,0 +1,7 @@
+const { runFastify } = require('./start')
+
+const fastify = runFastify(process.argv.splice(2))
+
+fastify.ready(err => {
+  process.send({ type: 'ready', err: err })
+})
```

Comment:
You might want to receive a signal about the parent a) closing and b) wanting to shut down this. Ideally we should shut fastify down gracefully.